lms/validation/blackboard.py

```diff
@@ -0,0 +1,26 @@
+"""
+Schemas for Blackboard API responses.
+
+See: https://developer.blackboard.com/portal/displayApi
+"""
+from marshmallow import INCLUDE, Schema, fields, post_load
+
+from lms.validation._base import RequestsResponseSchema
+
+
+class BlackboardListFilesSchema(RequestsResponseSchema):
+    """Schema for Blackboard API /courses/{courseId}/resources responses."""
+
+    class ResultsSchema(Schema):
+        class Meta:
+            unknown = INCLUDE
```

Comment:
The excluding unexpected things is just a case of using `EXCLUDE` here isn't it?

Reply:
Yep. Which is a lot more convenient than letting the schema include them and then the service having to iterate over the list of dicts from the schema in order to remove the fields that should have been excluded by the schema